CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
+
+All documentation, code and communication under this repository are covered by the [W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
```

Comment:
Can you rebase against master? I don't think you intended for this to be part of your PR.